### PR TITLE
Accumulate fractional moves across updates.  #5183

### PR DESCRIPTION
**Pull request by AMagill**
This change allows fractional pixel movements to accumulate between updates.

For example, if you have a DPI setting of 150% and move the mouse slowly to the right so that there is a one pixel movement every update, those movements will be scaled to 0.66 pixels.  Right now that value is always truncated to zero, making it difficult to move the mouse slowly down or right.

A more correct approach is to accumulate the fractional part of those movements.  For example:
Pixel movements:  +1, +1, +1, +1, +1, +1, +1, +1
DPI scaled movements:  +0.66, +0.66, +0.66, +0.66, +0.66, +0.66, +0.66, +0.66
Old movements sent to client:  0, 0, 0, 0, 0, 0, 0, 0
New movements sent to client: 0, 1, 1, 0, 1, 1, 0, 1

### CODE REVIEW

File: src/lib/platform/MSWindowsScreen.h [hunk 1]
```diff
@@ -266,6 +270,9 @@ class MSWindowsScreen : public PlatformScreen {
 	// last mouse position
 	SInt32				m_xCursor, m_yCursor;
 
+	// accumulated fractional pixel moves
+	float					m_xFractionalMove, m_yFractionalMove;
```
Comment (XinyuHou) [inline src/lib/platform/MSWindowsScreen.h]: Indentation to line 25 please

**Comment by XinyuHou** [comment]
@AMagill 
Could someone test this nightly?
http://synergy-project.org/nightly?filter=pr5190-accumulate-fractional-moves-beta-f240faf

**Comment by AMagill** [comment]
@XinyuHou Could you re-run that nightly build?

**Comment by XinyuHou** [comment]
Merged in 1.8.4

**Pull request closed without merging**